src/openrc-run/openrc-run.c

```diff
@@ -320,9 +323,9 @@ write_prefix(const char *buffer, size_t bytes, bool *prefixed)
 		}
 
 		if (!*prefixed) {
```

Comment:
this codepath is called once per-line.  while calling this function with multiline strings is certainly valid, and i'm sure done in places, it's not the common case, and I have a hard time seeing strlen being called on tiny strings showing up in any perf situation.

len_ec & ec_normal will be like 7 bytes each.  len_prefix will be ~length of longest service name, and that's likely to also be quite small.